packages/phone-number-privacy/test/utils.ts

```diff
@@ -1,5 +1,7 @@
 import BigNumber from 'bignumber.js'
+import threshold from 'blind-threshold-bls'
 import Web3 from 'web3'
+const btoa = require('btoa')
```

Comment:
Can we use es6 import syntax here? Should be
`import btoa from 'btoa'`
or in rare cases depending on the lib's export style:
`import * as btoa from 'btoa'`